modules/documents/client/directives/documents.manager.link.directive.js

```diff
@@ -248,19 +248,14 @@ angular.module('documents')
 
               self.ok = function () {
                 // return the data in the selected list...
-                $uibModalInstance.close({linked: self.linkedFiles, source : self.sourceDocs[self.docType], docType : self.docType});
+                $uibModalInstance.close({linked: self.linkedFiles, docType : self.docType/*, source : self.sourceDocs[self.docType]*/});
```

Comment:
Any reason to keep a commented-out parameter? It would be better if we removed it completely (same thing as line 258).